spec/prius/registry_spec.rb

```diff
@@ -3,7 +3,15 @@
 require "prius/registry"
 
 describe Prius::Registry do
-  let(:env) { { "NAME" => "Harry", "AGE" => "25", "ALIVE" => "yes" } }
+  let(:env) do
+    {
+      "NAME" => "Harry",
+      "AGE" => "25",
+      "ALIVE" => "yes",
+      "BORN" => "2022-09-02",
+      "FAVOURITE_DATE" => "2022-02-99"
```

Comment:
```suggestion
      "INVALID_DATE" => "2022-02-99"
```
?

Reply:
Yeah, I was going for the "variables about a person" theme, but `favourite_date` was pushing understandability a bit.